flow/scripts/global_place.tcl

```diff
@@ -9,6 +9,10 @@ fast_route
 
 set global_placement_args {}
 
+if {[info exists ::env(GPL_ALLOW_REVERT_IF_DIVERGE)] && $::env(GPL_ALLOW_REVERT_IF_DIVERGE)} {
+  lappend global_placement_args {-allow_revert_if_diverge}
```

Comment:
Use [append_env_var](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/b9b7f04809853355e5122996e9a287c6d7a32538/flow/scripts/detail_route.tcl#L12) for consistency and to avoid repeating tiny bugs